JS-Algorithms/challenges.js

```diff
@@ -118,7 +156,19 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let num1 = 0;
```

Comment:
I think you can use just two variables for this, also let's talk about variable naming, I would like you to bear in mind not use this kind of names num1, num2, you could use firstNumber or secondNumber instead